test/NaCl.Core.SimdTests/ChaCha20IntrinsicsTests .cs

```diff
@@ -0,0 +1,347 @@
+﻿namespace NaCl.Core.SimdTests
```

Comment:
We could switch to scoped namespace declaration here since we already using it on the other files =)